qutip/wigner.py

```diff
@@ -35,8 +35,9 @@
            'spin_wigner', 'wigner_transform']
 
 import numpy as np
-from numpy import (zeros, array, arange, exp, real, conj, pi,
-                   copy, sqrt, meshgrid, size, conjugate,
+import warnings
+from scipy import (zeros, array, arange, exp, real, conj, pi,
```

Comment:
Most of these should be imported from `numpy` not `scipy`. As of scipy 1.4 they raise a [deprecation warning](https://scipy.github.io/devdocs/release.1.4.0.html#deprecated-features) on use and will be removed in scipy 2.0.

Reply:
@dweigand can you do this simple change? Then we could merge if it is ok for all.